tests/test_timeouts.py

```diff
@@ -13,6 +15,7 @@ async def test_read_timeout(server):
 
 
 @pytest.mark.anyio
+@pytest.mark.xfail(sys.version_info >= (3, 14), reason="Fix trio on Python >= 3.14")
```

Comment:
Why is this test marked as xfail?

Reply:
Because it fails.

Reply:
Then it should fail, not being tolerated. Unless there's a valid rationale to mark as `xfail`

Reply:
You've marked this as "Fix trio on Python >= 3.14"...

- Can you confirm that you've observed this test hanging on `trio` and not on `asyncio`?
- Is there a relevant issue for this against trio?

Reply:
* Yes.  (Is there a way to write the xfail to only pertain to `trio`?)  -- `asyncio` passes.
* I will create one.